gruntfile.js

```diff
@@ -14,28 +14,45 @@ module.exports = function (grunt) {
 
     var concatConfig = {
         dist: {
-            src: [
-                'src/**/*.js',
-            ],
-            dest: 'dist/angular-token-auth.js',
+            files: [{
+                src: [
+                    '<%= config.files.src %>',
+                ],
+                dest: 'dist/angular-token-auth.js',
+            }],
         },
     };
 
     var uglifyConfig = {
         dist: {
+            options: {
+                screwIE8: false,
```

Comment:
See https://github.com/mishoo/UglifyJS2/compare/v2.6.2...v2.8.3#diff-04c6e90faac2675aa89e2176d2eec7d8

```diff

-  --screw-ie8                   Pass this flag if you don't care about full
-                                compliance with Internet Explorer 6-8 quirks
-                                (by default UglifyJS will try to be IE-proof).
+  --screw-ie8                   Use this flag if you don't wish to support
+                                Internet Explorer 6/7/8.
+                                By default UglifyJS will not try to be IE-proof.
```

Before, by default it would not change `promise['finally']` or `http['delete']` to `promise.finally` and `http.delete`. Now it does, but I don't feel a need to actively remove that support and I see no harm in keeping it, so turning off `--screw-ie8` is a good change to lessen the diff on the distribution.